README.md

```diff
@@ -40,6 +40,28 @@ Some main modules are:
 - [Metis](applications/MetisApplication/README.md)
 - [Meshing](applications/MeshingApplication/README.md)
 
+# Documentation
+Here you can find the basic documentation of the project:
+## Getting Started
+* Getting Kratos (Last compiled Release)
+    * [Kratos for Linux](https://github.com/KratosMultiphysics/Kratos/wiki/Getting-Kratos-Binaries-for-Linux)
+    * [Kratos for Windows](https://github.com/KratosMultiphysics/Kratos/wiki/Getting-Kratos-Binaries-for-Windows)
```

Comment:
I'd point to INSTALL.md in here

Reply:
This is precompiled version

Reply:
I know. But I'd point to how to install too. In any case this is just a temporary solution (I assume that once we have the final documentation platform we would only point there).

Reply:
But the install is in the line 52